homework5/task_4.py

```diff
@@ -0,0 +1,27 @@
+# Реализуйте RLE алгоритм: реализуйте модуль сжатия данных.
+# Входные и выходные данные хранятся в отдельных текстовых файлах.
+
+with open('RLE_decoded.txt', 'r') as file:
+    my_text = file.read()
+
+
+def encode_rle(ss):
+    str_code = ''
+    prev_char = ''
+    count = 1
+    for char in my_text:
+        if char != prev_char:
+            if prev_char:
+                str_code += str(count) + prev_char
+            count = 1
+            prev_char = char
+        else:
+            count += 1
+    return str_code
```

Comment:
У вас функция принимает аргумент `ss`, но не использует его. наверно вы хотели сделать так
```suggestion
def encode_rle(ss):
    str_code = ''
    prev_char = ''
    count = 1
    for char in ss:
        if char != prev_char:
            if prev_char:
                str_code += str(count) + prev_char
            count = 1
            prev_char = char
        else:
            count += 1
    return str_code
```